metadata/src/main/java/org/apache/kafka/controller/ConfigurationControlManager.java

```diff
@@ -374,6 +389,20 @@ void deleteTopicConfigs(String name) {
         configData.remove(new ConfigResource(Type.TOPIC, name));
     }
 
+    ConfigResource currentNodeResource() {
```

Comment:
Could this be private?

Reply:
No, this method is used to access the private variable `currentNodeResource` from outside of the class.